features/backend/products.feature

```diff
@@ -159,6 +156,18 @@ Feature: Products
         Then I should be on the page of product "Big Sticker"
         And I should see "Product has been successfully updated."
 
+    @javascript
+    Scenario: Modifying product attributes
+        Given I am editing product "Super T-Shirt"
+        When I go to "Attributes" tab
+        And I delete "T-Shirt fabric" attribute
+        And I add "Size" attribute
+        And I fill in "Size" with "100"
+        When I press "Save changes"
```

Comment:
To be honest I am always not sure, ``Go to`` is an action already. The thing is that this whole scenario should be something like:

``` gherkin
Given I want to edit product "Super T-Shirt"
When I remove "Fabric" attribute
And I add "Size" attribute with value "100"
And I save my changes
Then the product should have "Size" attribute with value "100"
```

Still far from perfect... Anyway, we don't want to change these now so I'd say it is fine like that.